common/src/main/java/io/homeassistant/companion/android/database/AppDatabase.kt

```diff
@@ -483,6 +487,12 @@ abstract class AppDatabase : RoomDatabase() {
             }
         }
 
+        private val MIGRATION_23_24 = object : Migration(23, 24) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                database.execSQL("ALTER TABLE `settings` ADD `sensorUpdateFrequency` TEXT NOT NULL DEFAULT ${SensorUpdateFrequencySetting.NORMAL}")
```

Comment:
They sugest migrations not depend on code at all in case our next build changes that constant.  Also need `'`.
```suggestion
                database.execSQL("ALTER TABLE `settings` ADD `sensorUpdateFrequency` TEXT NOT NULL DEFAULT 'NORMAL'")
```